homeassistant/components/apsystems_local/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""The APsystems local API integration."""
+
+from __future__ import annotations
+
+import logging
+
+from APsystemsEZ1 import APsystemsEZ1M
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_IP_ADDRESS, Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import ApSystemsDataCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up this integration using UI."""
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
Instead of this, use `entry.runtime_data` like in https://github.com/home-assistant/core/pull/117261/files